src/single-spa-react.js

```diff
@@ -150,3 +150,16 @@ function chooseDomElementGetter(opts, props) {
     return opts.domElementGetter
   }
 }
+
+function reactDomRender({opts, elementToRender, domElement, whenFinished}) {
+  if(opts.renderType === 'createRoot') {
+    return opts.ReactDOM.createRoot(domElement).render(elementToRender, whenFinished)
```

Comment:
Isn't this the undocumented API? Are we confident enough to support it now or should we wait?

Reply:
It's not in the official docs, but it's all over the source code and in all the examples that they've given on concurrent react. https://github.com/facebook/react/search?q=createRoot&unscoped_q=createRoot